### DISCUSSION TITLE
How to interpolate from a 3D flexible mesh (geometry: Flexible Mesh Geometry: Dfsu3DSigma)?

### DISCUSSION THREAD

**Discussion by Userpython18** [DHI/mikeio#639] [General]
I have a geometry: Flexible Mesh Geometry: Dfsu3DSigma. I want to interpolate to another mesh. But it shows an error when I use interp_like. (Currently only supports interpolating from 2d flexible mesh data!).

**Comment by ecomodeller** [comment]
There is no functionality for interpolation to another 3d mesh.

Feel free to add it as a feature request.

Is the use case to be able to compare two outputs from two different meshes?

**Reply by Userpython18** [reply]
Yes that's correct, it is to compare the output from two different meshes.

**Reply by ecomodeller** [reply]
Seems like a common use case. I wish we would have this kind of functionality.

**Comment by jsmariegaard** [comment]
@JesperGr and @floodydhi - it would be great to wrap core MIKE functionality for cases like this e.g. from MzChart !
